docs/components/toaster.md

```diff
@@ -2,65 +2,73 @@
 
 KToaster a popup notification component. The toaster will close automatically after a set timeout or can be dismissed by clicking on the close icon.
 
-KToaster is used via the `ToastManager` instance. All rendering is controlled from ToastManager via an intuitive, imperative API. It is recommended that you initialize `ToastManager` in your app via [`app.config.globalProperties`](https://vuejs.org/api/application.html#app-config-globalproperties) to allow you to access it on any component instance inside your application.
+KToaster is used via the `ToastManager` instance. All rendering is controlled from `ToastManager` via an intuitive API.
 
-```ts
-// main.ts
-
-import { createApp } from 'vue'
-import { ToastManager } from '@kong/kongponents'
+The easiest way to use it is by creating a composable that you can use anywhere in your app. This way you don't have to initialize `ToastManager` in every component.
 
-const app = createApp()
+```ts
+// composables/useToaster
 
-// Available inside any component template in the application, and also on 'this' of any component instance
-app.config.globalProperties.$toaster = new ToastManager({
-  // you can also override the default value of following properties while initializing ToastManager
-  id: 'custom-id', // default: 'toaster-container'
-  timeout: 500, // default: 5000
-  appearance: 'success', // default: 'info'
-  zIndex: 92929, // default: 10000
-})
-```
+import { onBeforeUnmount } from 'vue'
+import { ToastManager } from '@/index'
```

Comment:
issue: for the example, they will need to import from the package rather than a local file
```suggestion
import { ToastManager } from '@kong/kongponents'
```